docs/adr/0016-updating-non-wmde-software.md

```diff
@@ -0,0 +1,39 @@
+# 16) Updating the software not maintained by WMDE {#adr_0016}
+
+Date: 2021-05-05
+
+## Status
+
+proposed
+
+## Context
+
+Wikibase Suite includes a number of software components that are not maintained by Wikimedia Deutschland (WMDE). Those include (examples as of May 2021):
+- Software which is maintained by the Wikimedia Foundation (WMF), e.g. Wikidata Query Service (WDQS); in case of docker images also MediaWiki software.
+- Broadly-used software which is maintained by non-Wikimedia parties, e.g. Elastic Search, MySQL.
```

Comment:
Our suite doesn't actually include MySQL (or Maria), only our example docker-compose files include it.

Reply:
good catch, thanks!